unfoldingWord/translationCore
# Added User Card

#### This pull request addresses:
This PR adds all implementation for the user card section. #1946 


#### How to test this pull request:
Test the functionality listed in the issue above with the actual functionality. Also check against https://xd.adobe.com/view/105373ea-fd68-4cf5-8f07-61a3b67d4850/screen/96e44509-6925-40d6-8351-86e4162ba846/Local-Account-2
and #968 #966 #964


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1953)
